docs/milo/manual/chapters/20_introduction.md

```diff
@@ -22,62 +22,62 @@ Have fun building your very first Milo!
 
 ## What To Expect
 
-This manual and associated firmware is for the Milo V1.5 with a Mellow Fly CDYv3 control board installed in the internal electronics enclosure found in the machine base.
+This manual and associated firmware are for the Milo V1.5 with a Mellow Fly CDYv3 control board installed in the internal electronics enclosure found in the machine base.
 
 This manual will not cover external enclosures or the electronics compartment found in the Millennium Machines Casa enclosure - but where possible, it can still be used as a guideline for those other setups.
 
 The firmware used in this manual will only consist of the basic 3-axis functions of the machine.
 
-Touch probes, additional relays, toolsetters and other related add ons will not be part of this base firmware and therefore will not be present in this manual.
```

Comment:
"Not present" seems clearer

Reply:
This was addressed, however it didn't mark the request as outdated.